experimental/apps-mcp/lib/providers/databricks/catalogs.go

```diff
@@ -0,0 +1,157 @@
+package databricks
```

Comment:
Non-blocking: everything in this package, perhaps minus sql.go, seems like it should be removed and based with CLI-based logic.